test/cases/api.sh

```diff
@@ -438,6 +438,36 @@ EOF
 )
 export FIREWALL_CUSTOMIZATION_BLOCK
 
+if [ "$TEST_MODULE_HOTFIXES" = 1 ]; then
```

Comment:
Maybe I'm blind, but I don't see the `TEST_MODULE_HOTFIXES` being set anywhere :thinking:

Reply:
You're right, I need to set it in the gitlab test config

Reply:
Done, but I dunno if correctly xD

Reply:
Done :+1: